src/foraging_gui/Foraging.py

```diff
@@ -4977,6 +4990,16 @@ def log_git_hash():
     return git_hash, git_branch, repo_url, repo_dirty_flag, dirty_files, version
 
 
+def show_msg_box(window_title, title, msg):
```

Comment:
Seems odd to make a function that is only used once. I would prefer to not make a function here. But can you at least add a doc string